data_structures/src/chain.rs

```diff
@@ -1719,6 +1719,22 @@ impl TransactionsPool {
             .remove(key)
             .map(|(weight, transaction)| {
                 self.sorted_vt_index.remove(&(weight, *key));
+                // Remove the inputs from the output_pointer_map
```

Comment:
Should you repeat in the dr_remove_inner method?

Reply:
I had a test for DR transactions but removed it because "it should be the same". So that was a mistake :laughing: